config.properties

```diff
@@ -0,0 +1,21 @@
+title = RAINBOW BOOKSTORE
```

Comment:
you need to replace all occurences of RAINBOW BOOKSTORE in the code for this property. I found 12 matches. Please change it

Reply:
ok